core/msg.c

```diff
@@ -25,6 +25,7 @@
 #include <assert.h>
 #include "sched.h"
 #include "msg.h"
+#include "msg_bus.h"
```

Comment:
Not sure if this needs it's own header file or if it should simply be moved into `msg.h`